server/config/api.js

```diff
@@ -69,7 +69,8 @@ exports.default = {
 exports.test = {
   general: function (api) {
     return {
-      id: 'test-server',
+      id: `test-server-${process.pid || 0}`,
+      serverToken: `serverToken-${process.pid || 0}`,
```

Comment:
jest provides `JEST_WORKER_ID` which might be the better approach, or we may combine both just in case

Reply:
JEST_WORKER_ID is undefined when using jest-runner-mocha